merge-all.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+
+# Usage:
+# "REPOS_PATH=infrastructure GH_ACCESS_TOKEN=abc123 ./merge-all.rb"
+
+BRANCH_NAME = 'update-dependencies'
+
+def ruby_repos_file
+  File.open("#{ENV['REPOS_PATH']}/ruby")
+end
+
+def repos
+  [ruby_repos_file].map(&:to_a).flatten.map(&:strip).sort
+end
+
+# @return [Array<Hash>] the update PR
+def find_prs(client, repos)
+  repo_content = repos.map do |repo|
+    prs = client.pull_requests(repo, head: "sul-dlss:#{BRANCH_NAME}")
```

Comment:
There's a lot of `prs.first` below. We expect this to be an array of length 1 every time, no? (A length of zero is possible but not very likely...)

Reply:
Good suggestion.